tdrs-backend/tdpservice/users/test/test_serializers.py

```diff
@@ -1,34 +1,29 @@
 """Test user serializers."""
-
-from django.test import TestCase
-from django.forms.models import model_to_dict
 from django.contrib.auth.hashers import check_password
-from nose.tools import eq_, ok_
-from .factories import UserFactory
+
+import pytest
+
 from ..serializers import CreateUserSerializer
 
 
-class TestCreateUserSerializer(TestCase):
```

Comment:
It looks like some of our tests are class based and others function based. Is there a reason to utilize both, or might we want to adopt one style for the sake of consistency?

Reply:
Im open to either,  this was a result of me following examples and practices for testing to get our coverage up.